<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetsEditingObjectSaveCancelButtons</class>
 <widget class="QWidget" name="WidgetsEditingObjectSaveCancelButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="BtnDiff">
     <property name="text">
      <string>Diff</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="BtnSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="BtnCancel">
     <property name="text">
      <string>Discard</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
